nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -124,6 +133,9 @@ public Description createErrorDescription(
       builder = addSuggestedSuppression(errorMessage, suggestTree, builder);
     }
     // #letbuildersbuild
+    if (config.getAutoFixConfig().LOG_ERROR_ENABLED) {
```

Comment:
I assume this is false whenever autofix is `false`, but can we gate these by `config.autofixIsEnabled() && ...` to make it clearer?

Reply:
Yes, all flags in ```AutoFixConfig``` are set with the formula below:

```java
flag = flags.get("flag") && autoFixEnabled
```
However, I still added the guard you suggested.